cmd/generic-handlers.go

```diff
@@ -470,6 +470,12 @@ func (h resourceHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	// If bucketName is present and not objectName check for bucket level resource queries.
 	if bucketName != "" && objectName == "" {
+		// enable GetBucketACL dummy call specifically.
+		_, ok := r.URL.Query()["acl"]
+		if r.Method == http.MethodGet && ok {
+			h.handler.ServeHTTP(w, r)
+			return
+		}
```

Comment:
suggestion: ignoreNotImplementedBucketResources() and ignoreNotImplementedObjectResources() is a better place to abstract this detail.

i.e
```
@@ -420,6 +420,9 @@ func setIgnoreResourcesHandler(h http.Handler) http.Handler {
 // Checks requests for not implemented Bucket resources
 func ignoreNotImplementedBucketResources(req *http.Request) bool {
        for name := range req.URL.Query() {
+               if name == "acl" && req.Method == http.MethodGet {
+                       return false
+               }
                if notimplementedBucketResourceNames[name] {
                        return true
                }
@@ -430,6 +433,9 @@ func ignoreNotImplementedBucketResources(req *http.Request) bool {
 // Checks requests for not implemented Object resources
 func ignoreNotImplementedObjectResources(req *http.Request) bool {
        for name := range req.URL.Query() {
+               if name == "acl" && req.Method == http.MethodGet {
+                       return false
+               }
                if notimplementedObjectResourceNames[name] {
                        return true
                }
```

Reply:
Yeah let me add it there, I only wanted to implement this for buckets, not objects.